pkg/apps/blackduck/latest/containers/postgres.go

```diff
@@ -81,16 +80,10 @@ func (c *Creater) GetPostgres() *postgres.Postgres {
 	}
 
 	if !c.config.IsOpenshift { // TODO: Support SecurityContexts for Openshift in the next release
-		postgresSecurityContext := apputil.GetSecurityContext(c.blackDuck.Spec.SecurityContexts, "blackduck-postgres")
-		if postgresSecurityContext == nil {
-			postgresSecurityContext = &api.SecurityContext{
-				FsGroup:    util.IntToInt64(26),
-				RunAsUser:  util.IntToInt64(26),
-				RunAsGroup: util.IntToInt64(26),
-			}
+		if postgresSecurityContext := apputil.GetSecurityContext(c.blackDuck.Spec.SecurityContexts, "blackduck-postgres"); postgresSecurityContext != nil {
+			utilPostgresSecurityContext := util.SecurityContext(*postgresSecurityContext)
+			postgresConfig.SecurityContext = &utilPostgresSecurityContext
```

Comment:
Just wondering why this can't be made like the other pods now?

Reply:
Uses the same logic now :)